test/test_torch.py

```diff
@@ -11867,6 +11867,38 @@ def test_linspace_vs_numpy(self, device, dtype):
             self.assertTrue(t[0] == a[0])
             self.assertTrue(t[steps - 1] == a[steps - 1])
 
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    @precisionOverride({torch.cfloat: 1e-6, torch.cdouble: 1e-10})
+    @dtypes(torch.cfloat, torch.cdouble)
+    @onlyCPU
+    def test_linspace_vs_numpy_complex(self, device, dtype):
+        start = torch.randn(1, dtype=dtype).item()
+        end = (start + torch.randn(1, dtype=dtype) + random.randint(5, 15)).item()
+
+        def test_fn(torch_fn, numpy_fn, steps):
+            t = torch_fn(start, end, steps, device=device, dtype=dtype)
+            a = numpy_fn(start, end, steps, dtype=torch_to_numpy_dtype_dict[dtype])
+            t = t.cpu()
+            self.assertEqual(t, torch.from_numpy(a))
```

Comment:
Haven't added `self.assertTrue(t[0] == a[0])` as 
```python
>>> import torch
>>> import numpy as np
>>> t = torch.tensor([10j], dtype=torch.complex64)
>>> a = np.array([10j], dtype=np.complex64)
>>> t[0] == a[0]
False
```

Reply:
yeah that's because `t[0]` returns a 0 dimensional tensor as opposed to numpy which returns the value contained inside the array.